Examples/VimeoUpload-iOS/VimeoUpload-iOS/NewVimeoUploader.swift

```diff
@@ -25,17 +25,18 @@
 //
 
 import Foundation
+import VimeoUpload
 
-class OldVimeoUpload: VimeoUpload<OldUploadDescriptor>
+class NewVimeoUploader: VimeoUploader<UploadDescriptor>
```

Comment:
I'm really confused by this, I'm hoping it's just github's difftool being weird.  It looks like the NewVimeoUpload class was renamed to OldVimeoUploader and the OldVimeoUpload class was renamed to NewVimeoUploader.  Is that right?